geode-core/src/test/java/org/apache/geode/util/test/TestUtil.java

```diff
@@ -46,11 +47,15 @@ public static String getResourcePath(Class<?> clazz, String name) {
         File tmpFile = File.createTempFile(filename, null);
         tmpFile.deleteOnExit();
         FileUtil.copy(resource, tmpFile);
-        return tmpFile.getAbsolutePath();
+        return compatibleWithWindows(tmpFile.getAbsolutePath());
       }
-      return path;
+      return compatibleWithWindows(path);
     } catch (URISyntaxException | IOException e) {
       throw new RuntimeException("Failed getting path to resource " + name, e);
     }
   }
+
+  private static String compatibleWithWindows(String path) {
```

Comment:
I'd prefer to see it private to minimize tests using this util class. Util classes are the bane of well-organized tests. You end up with 30 tests using the same util class and by then it's like a whole product of its own and the tests don't even look like they're testing the product code anymore.

My advice: keep this as private, move the other method into the test class and make it private too.